crates/sui-types/src/sui_serde.rs

```diff
@@ -181,6 +189,32 @@ impl Base64 {
     }
 }
 
+pub struct SuiBitmap;
```

Comment:
Here I would emphasize and link to the [roaring bitmap on-disk format](https://github.com/RoaringBitmap/RoaringFormatSpec).